shared/src/commonMain/kotlin/io.newm.shared/public/models/WalletConnection.kt

```diff
@@ -5,6 +5,8 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class WalletConnection(
     val id: String,
+    // val localName: String, do we want to add the custom name here?
+    // val blockchainType: String, how we choose the wallet icon
```

Comment:
Yes, we have a ticket on the backend side to add both of these items to the wallet connection response 
cc: @wlara